guides/v2.3/graphql/reference/quote-apply-giftcard.md

```diff
@@ -0,0 +1,76 @@
+---
+group: graphql
+title: applyGiftCardToCart mutation
+ee_only: True
+---
+
+The `applyGiftCardToCart` mutation applies a pre-defined gift card code to the specified cart.
+
+## Syntax
+
+ `mutation: applyGiftCardToCart(input: ApplyGiftCardToCartInput): ApplyGiftCardToCartOutput`
+
+## Example usage
+
+ The following example adds a gift card code called `048FQEGYUA73` to a cart.
+
+**Request**
+
+ ``` text
+mutation {
+   applyGiftCardToCart(
+      cart_id: "IeTUiU0oCXjm0uRqGCOuhQ2AuQatogjG"
+      gift_card_code: "048FQEGYUA73"
+   ) {
+    cart {
+    applied_gift_cards {
```

Comment:
Indent `applied_gift_cards` by a few space. Also, `code`.  The overall alignment might be off a bit. The last } doesn't look flush left.